agent/acs/model/ecsacs/api.go

```diff
@@ -285,6 +285,44 @@ func (s AttachTaskNetworkInterfacesOutput) GoString() string {
 	return s.String()
 }
 
+type Attachment struct {
+	_ struct{} `type:"structure"`
+
+	AttachmentArn *string `locationName:"attachmentArn" type:"string"`
+
+	AttachmentProperties []*AttachmentProperty `locationName:"attachmentProperties" type:"list"`
```

Comment:
nit: You dont need to use a pointer for list objects. You could simply check length of the list.